app/graphql/types/mutation_type.rb

```diff
@@ -1,5 +1,5 @@
 module Types
-  # class MutationType < Types::BaseObject
```

Comment:
Tested manually, team is planning to write tests later so that FE can start testing with it on their end.